health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/index.js

```
@@ -275,16 +274,15 @@ function DeliverySetup({ onSelect, config, formData, control, tabCount = 2, subT
-                products: [],
-              },
-            ];
+        const updatedDeliveryRules = [
+          ...action.payload.currentDeliveryRules,
+          {
+            ruleKey: action.payload.currentDeliveryRules.length + 1,
+            delivery: {},
+            attributes: [{ key: 1, attribute: null, operator: null, value: "" }],
+            products: [],
+          },
+        ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)